trulens/nn/models/keras_utils.py

```
@@ -250,7 +278,12 @@ def tracer(depth):
                     innode_indices[out_layer.name] = idx
         tracer(depth + 1)
 
-    tracer(0)
+    try:
+        innode_indices = {}
+        nodes_by_depth = model._nodes_by_depth
+        tracer(0)
+    except AttributeError:
+        return {}
```

Can we log what this might be instead of silent fail?